vscode-extension/README.md

```diff
@@ -1,71 +1,51 @@
 
-If you have any requirements or dependencies, add a section describing those and how to install and configure them.
+### Quick Start
 
-## Extension Settings
+1. Install the [AIConfig Editor extension](https://marketplace.visualstudio.com/items?itemName=lastmile-ai.vscode-aiconfig).
+2. Create a .env file with your API keys: e.g. `OPENAI_API_KEY='your-key’`
+3. Open Command Palette (`Ctrl`+`Shift`+`P`) and select `AIConfig: Create New` to launch an Untitled aiconfig.
 
-Include if your extension adds any VS Code settings through the `contributes.configuration` extension point.
+## Key Features
 
-For example:
+- **Stay in flow**. Access all generative AI models in a single place, directly in your IDE.
+- **Universal notebook playground**. Swap between models, chain models together, and create prompt templates.
+- **Access local and remote models in one place**. Out-of-the-box support for text, image, and audio models.
+- **Version control Prompts**. Manage prompts right beside your code.
+- **Connect to your own models and endpoints**. [Extend AIConfig](https://aiconfig.lastmileai.dev/docs/extensibility) to work with any model and endpoint.
 
-This extension contributes the following settings:
+## Supported Models
 
-- `myExtension.enable`: Enable/disable this extension.
-- `myExtension.thing`: Set to `blah` to do something.
+This extension supports all major foundation models from major model providers.
 
-## Known Issues
+For more details, please see https://aiconfig.lastmileai.dev/docs/overview/model-parsers/.
 
-Calling out known issues can help limit users opening duplicate issues against your extension.
-
-## Release Notes
-
-Users appreciate release notes as you update your extension.
-
-### 1.0.0
-
-Initial release of ...
-
-### 1.0.1
-
-Fixed issue #.
-
-### 1.1.0
-
-Added features X, Y, and Z.
-
----
-
-## Following extension guidelines
-
-Ensure that you've read through the extensions guidelines and follow the best practices for creating your extension.
-
-- [Extension Guidelines](https://code.visualstudio.com/api/references/extension-guidelines)
-
-## Working with Markdown
-
-You can author your README using Visual Studio Code. Here are some useful editor keyboard shortcuts:
-
-- Split the editor (`Cmd+\` on macOS or `Ctrl+\` on Windows and Linux).
-- Toggle preview (`Shift+Cmd+V` on macOS or `Shift+Ctrl+V` on Windows and Linux).
-- Press `Ctrl+Space` (Windows, Linux, macOS) to see a list of Markdown snippets.
-
-## For more information
-
-- [Visual Studio Code's Markdown Support](http://code.visualstudio.com/docs/languages/markdown)
-- [Markdown Syntax Reference](https://help.github.com/articles/markdown-basics/)
-
-**Enjoy!**
+| Provider                          | Model                                                                                                        | Language | Support                                                                                                     |
```

Comment:
That's true -- the main question is this readme displays in the extension marketplace so it might be useful to still keep, but agreed that it's pretty annoying to maintain two places. Will ship as is but we can maybe reword this to make it more generic, with a link to specific models supported